newsPaper.html

```diff
@@ -0,0 +1,117 @@
+      }
+    }
+    </script>
+
+  </head>
+
+  <body>
+    <div class = "jumbotron">
+      <div class ="container">
+        <div class ="row justify-content-center">
+          <div class ="col-md-4 ml-auto">
+                <h2> School start in Hamburg: mask requirement during breaks
+                </h2>
+              <p>
+                  At the start of the school after the summer vacation on Thursday,
+                 <span id="dots">...</span>
+                 <span id="more">
+                  Hamburg's schoolchildren
+                  and teachers in the buildings of secondary
+                  schools should now wear mouth-nose protection - at least outside of class.
+                  Hamburg's school senator Ties Rabe (SPD) said this on Monday morning in
+                  an interview with NDR 90.3.
+                </span>
+              </p>
+              <button onclick="myFunction()" id="myBtn">Read more</button>
+          </div>
+
+          <div class ="col-md-4 ml-auto">
+                <h2>Mask only when sitting down
+                </h2>
+              <p>
+                  Only when the students sit down in the classrooms at their place
+                   of work or <span id="dots">...</span>
+                   <span id="more">
+                   when teachers sit at their place of work does the mask
+                   requirement cease to apply. "But on the paths, in the school corridors,
+                   in the breaks, the masks are to be worn without exception," said Rabe.
+                   However, this does not apply to elementary school students up to ten
+                   years of age. The risk of infection is significantly lower for them.
+                   In addition, they could not yet handle the masks properly.
+                   </span>
+                </p>
+                <button onclick="myFunction()" id="myBtn">Read more</button>
+          </div>
+
+          <div class ="col-md-4 ml-auto">
+                <h2>Talks on the weekend
+                </h2>
+              <p>
+                "We coordinated with leading associations, virologists and experts
+                <span id="dots">...</span>
+                <span id="more"> over the weekend, who have now come to the conclusion that a mask
+                requirement makes sense," said Rabe. "That's why Hamburg will introduce
+                such a mask requirement - for everyone who is at school."
+                </span>
+              </p>
+              <button onclick="myFunction()" id="myBtn">Read more</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  <!--<SCRIPT TYPE="text/javascript"> function popup(mylink, windowname)
+    { if (! window.focus)return true; var href; if (typeof(mylink) == 'string')
+    href=mylink; else href=mylink.href; window.open(href, windowname, 'width=400,height=200,scrollbars=yes'); return false; }
+  </SCRIPT>
+  <BODY onLoad="popup('creeper.jpg', 'ad')">-->

```

Comment:
Will you need this commented code later?

Reply:
Maybe I do but I can remove it from here.